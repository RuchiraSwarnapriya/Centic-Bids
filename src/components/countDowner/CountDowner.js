import React from 'react';
import CountDown from 'react-native-countdown-component';

const CountDowner = ({remaningTime, onFinish, size}) => {
    return (
        <CountDown
            until={remaningTime}
            onFinish={onFinish}
            size={size}
        />
    )
}

export default CountDowner
