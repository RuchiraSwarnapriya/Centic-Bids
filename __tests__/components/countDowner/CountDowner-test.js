import React from 'react';
import renderer from 'react-test-renderer';
import CountDowner from '../../../src/components/countDowner/CountDowner';

jest.useFakeTimers();

test('renders correctly', () => {
  const tree = renderer.create(<CountDowner />).toJSON();
  expect(tree).toMatchSnapshot();
});