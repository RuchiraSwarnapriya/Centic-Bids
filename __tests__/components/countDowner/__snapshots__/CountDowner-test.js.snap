// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "center",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderRadius": 5,
                "justifyContent": "center",
                "marginHorizontal": 2,
              },
              Object {
                "height": 39,
                "width": 34.5,
              },
              Object {
                "backgroundColor": "#FAB913",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "white",
                  "fontVariant": Array [
                    "tabular-nums",
                  ],
                  "fontWeight": "bold",
                },
                Object {
                  "fontSize": 15,
                },
                Object {
                  "color": "#000",
                },
              ]
            }
          >
            00
          </Text>
        </View>
      </View>
      <Text
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "marginVertical": 2,
            },
            Object {
              "fontSize": 8.333333333333334,
            },
            Object {
              "color": "#000",
            },
          ]
        }
      >
        Days
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "center",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderRadius": 5,
                "justifyContent": "center",
                "marginHorizontal": 2,
              },
              Object {
                "height": 39,
                "width": 34.5,
              },
              Object {
                "backgroundColor": "#FAB913",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "white",
                  "fontVariant": Array [
                    "tabular-nums",
                  ],
                  "fontWeight": "bold",
                },
                Object {
                  "fontSize": 15,
                },
                Object {
                  "color": "#000",
                },
              ]
            }
          >
            00
          </Text>
        </View>
      </View>
      <Text
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "marginVertical": 2,
            },
            Object {
              "fontSize": 8.333333333333334,
            },
            Object {
              "color": "#000",
            },
          ]
        }
      >
        Hours
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "center",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderRadius": 5,
                "justifyContent": "center",
                "marginHorizontal": 2,
              },
              Object {
                "height": 39,
                "width": 34.5,
              },
              Object {
                "backgroundColor": "#FAB913",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "white",
                  "fontVariant": Array [
                    "tabular-nums",
                  ],
                  "fontWeight": "bold",
                },
                Object {
                  "fontSize": 15,
                },
                Object {
                  "color": "#000",
                },
              ]
            }
          >
            00
          </Text>
        </View>
      </View>
      <Text
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "marginVertical": 2,
            },
            Object {
              "fontSize": 8.333333333333334,
            },
            Object {
              "color": "#000",
            },
          ]
        }
      >
        Minutes
      </Text>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "center",
          }
        }
      >
        <View
          style={
            Array [
              Object {
                "alignItems": "center",
                "borderRadius": 5,
                "justifyContent": "center",
                "marginHorizontal": 2,
              },
              Object {
                "height": 39,
                "width": 34.5,
              },
              Object {
                "backgroundColor": "#FAB913",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "white",
                  "fontVariant": Array [
                    "tabular-nums",
                  ],
                  "fontWeight": "bold",
                },
                Object {
                  "fontSize": 15,
                },
                Object {
                  "color": "#000",
                },
              ]
            }
          >
            00
          </Text>
        </View>
      </View>
      <Text
        style={
          Array [
            Object {
              "backgroundColor": "transparent",
              "color": "white",
              "marginVertical": 2,
            },
            Object {
              "fontSize": 8.333333333333334,
            },
            Object {
              "color": "#000",
            },
          ]
        }
      >
        Seconds
      </Text>
    </View>
  </View>
</View>
`;
